vmdb/app/models/configuration_tag.rb

```diff
@@ -0,0 +1,8 @@
+class ConfigurationTag < ActiveRecord::Base
+  include NewWithTypeStiMixin
+  include ReportableMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :manager
+end
```

Comment:
Should you have has_and_belongs_to_many relationships to point back to configuration_profile and configured_system?